csss-site/src/events/frosh/templates/frosh/2023/frosh.html

```diff
@@ -0,0 +1,257 @@
+			<h3>How can I join?</h3>
+			<p>
+				Please <b>Sign-Up</b> to secure your place in
+				RETRO FROSH. Attendees receive a free RETRO
+				FROSH themed T-Shirt, as well as several
+				free meals throughout the week.
+			</p>
+			<div class="spacer"></div>
+			<h3>Gallery</h3>
+			<div class="imgs">
+				<div class="img">
+					<img src="{% static 'frosh_static/2023/images/gallery1.png' %}" alt="Frosh Week 2021: Hike">
+					<p>Frosh Week 2021: Hike</p>
+				</div>
+				<div class="img">
+					<img src="{% static 'frosh_static/2023/images/gallery2.png' %}" alt="Frosh Week 2011: Midnight Madness">
+					<p>Frosh Week 2011: Midnight Madness</p>
+				</div>
+				<div class="img">
+					<img src="{% static 'frosh_static/2023/images/gallery3.png' %}" alt="Candy Frosh (2022): PacMacro">
+					<p>Candy Frosh (2022): PacMacro</p>
+				</div>
+				<div class="img">
+					<img src="{% static 'frosh_static/2023/images/gallery4.png' %}" alt="Candy Frosh (2022): Midnight Madness">
+					<p>Candy Frosh (2022): Midnight Madness</p>
+				</div>
+				<div class="img">
+					<img src="{% static 'frosh_static/2023/images/gallery5.png' %}" alt="Frosh Week 2018: BBQ">
+					<p>Frosh Week 2018: BBQ</p>
+				</div>
+				<div class="img">
+					<img src="{% static 'frosh_static/2023/images/gallery6.png' %}" alt="Frosh Week 2017: Midnight Madness">
+					<p>Frosh Week 2017: Midnight Madness</p>
+				</div>
+			</div>
+		</section>
+		<section>
+			<h2>Schedule</h2>
+			<div class="section-anchor" id="schedule"></div>
+			<p>
+				RETRO FROSH starts <b>Monday, September 11<sup>th</sup></b>
+				through to <b>Saturday, September 16<sup>th</sup></b>.
+			</p>
+			<p>
+				A wide variety of events are planned for RETRO FROSH.
+				You are not obligated to attend all of them, but
+				we would love for you to come out whenever possible!
+			</p>
+			<p>
+				Further information, updates, and notifications
+				about events will be shared via <b>email</b> and
+				<b>announcements</b>. You will gain access to
+				these notifications by signing up.
+			</p>
+			<img class="schedule" src="{% static 'frosh_static/2023/images/schedule.svg' %}" alt="Schedule">
+		</section>
+		<section>
+			<h2>Sign-Up</h2>
+			<div class="section-anchor" id="sign_up"></div>
+			<p>
+				By signing up, you will gain access to all of
+				the above described <b>events</b>, <b>food</b>,
+				as well as your very own RETRO FROSH <b>T-Shirt</b>.
+			</p>
+			<p>
+				Please fill out the Google form below to <b>Sign-Up</b>.
+			</p>
+			<button onclick="location.href='https://forms.gle/fLwTi15iZy8qnySPA'" class="rounded leader">
+				<img src="{% static 'frosh_static/2023/images/register.svg' %}" alt="Register">
+				Sign-Up
+			</button>
+		</section>
+		<section>
+			<h2>Frequently Asked Questions</h2>
+			<div class="section-anchor" id="faq"></div>
+			<h3>Frosh Week FAQ</h3>
+			<h4>Where will the events take place?</h4>
+			<p>All events take place at the <b>Simon Fraser
```

Comment:
i suggest adding `text-indent: 0; margin-left: 24px;`, to remove indents, and a `<br>` after all the responses for every question.

ex:

Reply:
(I concede maybe the spacing should be a little less significant)